CMake/Servus.cmake

```diff
@@ -1,6 +1,16 @@
 
 set(SERVUS_PACKAGE_VERSION 1.0)
-set(SERVUS_REPO_URL https://github.com/HBPVis/servus.git)
-set(SERVUS_DEPENDS OPTIONAL DNSSD avahi-client Boost)
+set(SERVUS_REPO_URL https://github.com/HBPVis/Servus.git)
+set(SERVUS_SOURCE "${CMAKE_CURRENT_SOURCE_DIR}/src/Servus")
+set(SERVUS_DEPENDS hbpvis REQUIRED Boost Threads OPTIONAL DNSSD avahi-client)
```

Comment:
Most probably config.bluebrain was not updated. I wrote the Servus.cmake file but I didn't updeted config.bluebrain.
Boost shouldn't be required because we don't want to enforce it to build in Windows with a modern version of MSVC that supports C++11.